design.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path')
+
+const icons = [
+  'view-grid',
+  'chevron-right',
+  'chevron-left',
+  'format-rotate-90',
+  'file-download',
+  'printer',
+  'magnify-plus-outline',
+  'magnify-minus-outline',
+  'file-document',
+  path.join(path.dirname(__filename), './public/core-symbol.svg'),
```

Comment:
I used `path.dirname()` below because resolving `package.json` is the safest and the only way to get the package root, especially if the package does not have any `main`.

```suggestion
  path.join(__dirname, './public/core-symbol.svg'),
```